keps/sig-node/3066-subsecond-probes/README.md

```diff
@@ -0,0 +1,653 @@
+fields that specify seconds for intervals.
+Some users would like to have intervals less than or slightly greater
+than one second.
+This KEP adds sub-second interval capablilities to Kubernetes Liveness,
+Readiness, and Startup Probes.
+
+
+### Test Plan
+
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+Unit tests, end-to-end tests, and performance tests/benchmarks will need to be added.
+
+For actual tests to be added, see [the draft implementation](https://github.com/kubernetes/kubernetes/pull/107958/commits)
+
+##### Unit tests
+
+Will add unit tests for new code, specifically in:
+
+- `pkg/api/pod`: `2022-10-03` - `70.1`
+- `pkg/apis/core/validation`: `2022-10-03` - `82.3`
+- `pkg/kubelet/prober`: `2022-10-03` - `80.7`
+
+##### Integration tests
+
+Not adding integration tests (will be covered by e2e). 
+
+##### e2e tests
+
+Will add e2e tests to `test/e2e/common/node/container_probe.go` to (a) run a sub-second probe, and (b) ensure the sub-second probe completes faster than a standard (1s) probe.
+
+
+### Graduation Criteria
+
+
+### Upgrade / Downgrade Strategy
+
+### Version Skew Strategy
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+_This section must be completed when targeting alpha to a release._
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [ X ] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: SubSecondProbes
+    - Components depending on the feature gate:
+  - [ ] Other
+    - Describe the mechanism:
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+
+* **Does enabling the feature change any default behavior?**
+  Any change of default behavior may be surprising to users or break existing
+  automations, so be extremely careful here.
+
+  No
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  Also set `disable-supported` to `true` or `false` in `kep.yaml`.
+  Describe the consequences on existing workloads (e.g., if this is a runtime
+  feature, can it break the existing applications?).
+
+  Yes, see the [drop test](https://github.com/psschwei/kubernetes/blob/1ca40771e26b96d6121c49b19c2cbe6466694c7a/pkg/api/pod/util_test.go#L1029)
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+
+  No breaking changes
+
+* **Are there any tests for feature enablement/disablement?**
+
+  Yes, see the [drop test](https://github.com/psschwei/kubernetes/blob/1ca40771e26b96d6121c49b19c2cbe6466694c7a/pkg/api/pod/util_test.go#L1029)
```

Comment:
This link seems to be stale, could we update this to reflect a valid link for review purposes?